GameData/SPEngine/Parts/Kerovac.cfg

```diff
@@ -34,38 +26,70 @@
 	}
 	@identicalParts = SPEngineLR91Kero,SPEngineMVac // we're not identical to the part we copied!
 	!engineType = // prevent ROE from deleting us
-	@MODULE[ModuleEngines*]
-	{
-		%powerEffectName = Kerolox-Upper
-	}
-	@PLUME
+	// delete TEATEB.  For some reason !RESOURCE[TEATEB] doesn't work
+	!RESOURCE,* {}
+}
+
++PART[engineLargeSkipper_125m]:NEEDS[!ROEngines]:FOR[SPEngine]
+{
+	@name = SPEngineMCVac
+	@title = V-class Procedural Engine (Alternate model)
+	@description = Kerolox vacuum engine, like LR91 family (Titan I) and later SpaceX Merlin Vac.  <b><color=green>From RealFuels mod</color></b>
+	@TechRequired = orbitalRocketry1998
+	MODULE
 	{
-		@name = Kerolox-Upper
+		name = ModuleSPEngine
+		familyLetter = V
 	}
-	// delete TEATEB.  For some reason !RESOURCE[TEATEB] doesn't work
+	@identicalParts = SPEngineLR91Kero,SPEngineMVac // we're not identical to the part we copied!
+	!engineType = // prevent ROE from deleting us
 	!RESOURCE,* {}
 }
 
-+PART[engineLargeSkipper_125m]:BEFORE[ZREALPLUME]
++PART[engineLargeSkipper_125m]:NEEDS[!ROEngines]:FOR[SPEngine]
 {
-	@name = SPEngineMVac
+	@name = SPEngineMDVac
```

Comment:
Renaming parts isn't a good idea, as it'll break existing saves and craft files.

If there was a `SPEngineMVac` before your changes, there should still be one after.